### DISCUSSION TITLE
Android App Icon

### DISCUSSION THREAD

**Discussion by DavidM29** [unoplatform/uno#19136] [Q&A] [answered]
Can I use adaptive icons, generated in android studio, in uno platform as android launcher icon?
I generated the icons and moved them over to Resources:
![image](https://github.com/user-attachments/assets/0a2c2ee1-1d02-454e-b951-c61029c79f86)

But I get this error: `resource mipmap/ic_launcher_foreground (aka ge.bit.businessassistantpos:mipmap/ic_launcher_foreground) not found.`

**Comment by DavidM29** [comment]
This particular error can be fixed by removing `mimmap-anydpi-v26 folder`, I guess the error was because it does not contain `ic_launcher_foreground`.

However, the problem now is that launcher icon on android is being overriden by icons in Uno Assets, if I delete those the project does not get built anymore, is there a way around this?

**Reply by morning4coffe-dev** [reply]
Hey @DavidM29, 

thank you for your question. For this you would need to disable Resizetizer asset generation, I don't think we currently have a direct way to do this with Uno.Sdk. I have opened an [issue](https://github.com/unoplatform/uno/issues/19139) for this, to track more details. In the meantime, what you can do is to disable implicit packages ```<DisableImplicitUnoPackages>true</DisableImplicitUnoPackages>``` and add manually the required ones, without Uno.Resizetizer and it should work. You can find more details about this [here](https://platform.uno/docs/articles/features/using-the-uno-sdk.html?tabs=uno-packages#implicit-packages). Let me know if this helps!

**Reply by DavidM29** [reply]
Thanks @morning4coffe-dev, I will try this as soon as I get the chance.

**Reply by DavidM29** [reply]
Hello again @morning4coffe-dev,
I was able to generate icons in `Package.appxmanifest` Visual Assets and I am using those as an app icon on windows.

My goal is to do the same thing on android by generating icons in android studio and then moving them into my project, it is possible in MAUI according to [this article](https://medium.com/@Cizzl/a-step-by-step-guide-to-setting-up-adaptive-icons-in-net-maui-for-android-252897a867a2).

Why does specifying platform specific icons work on windows and not android?
Could I be doing something wrong and resizetizer is not actually the issue?

**Reply by morning4coffe-dev** [reply]
Did you manage to disable Resizetizer? There are some comments from my colleague as well on the issue https://github.com/unoplatform/uno/issues/19139 which I created. I think the Icon gets overwritten with the assets created by Resizetizer with Android, otherwise you should be technically able to use the same method for icons as MAUI.

**Reply by DavidM29** [reply]
No, not yet, the deadline is soon and the android icon is not the main priority at the moment, I will post an update as soon as I am able to try it out.
Thanks for the help.

**Reply by MartinZikmund** [reply]
Overall, we will need to review how we handle icons/splash screens across targets and how to easily enable escape hatches so users can choose to go a native route instead if they decide so

**Reply by DavidM29** [reply]
Hello @MartinZikmund, is this going to be tracked somewhere so I can follow?

**Reply by MartinZikmund** [reply, marked as answer]
@DavidM29 Converted this to an issue here https://github.com/unoplatform/uno.resizetizer/issues/333
